__tests__/main.spec.ts

```diff
@@ -1255,6 +1255,49 @@ test('stale label should be removed if a comment was added to a stale issue', as
   expect(processor.removedLabelIssues).toHaveLength(1);
 });
 
+test('if specified, labels should be added when unstale', async () => {
+  const opts = {
```

Comment:
```suggestion
  expect.assertions(4);
  const opts = {
```
IMO it's a good practice for async tests to have this